frontend/src/components/ImplementationReportView/EmbedBadges.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import { Implementation } from "../../data/parseReportData";
+import CopyToClipboard from "../CopyToClipboard";
+import Dialect from "../../data/Dialect";
+import { complianceBadgeFor, versionsBadgeFor } from "../../data/Badge";
+import { badgeFormatOptions, BadgeFormatOption } from "./BadgeFormats";
+
+const EmbedBadges: React.FC<{
+  implementation: Implementation;
+}> = ({ implementation }) => {
+  const [activeTab, setActiveTab] = useState("URL");
```

Comment:
Let's focus the Markdown tab to start, as for better or worse that's the most common README format.